covid19datavis.md

```diff
@@ -0,0 +1,32 @@
+# COVID 19 Data Exploration and Visualization
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+{{COVID 19 Data Exploration and Visualization}}
+
+**TOPIC:**    
+{{Probability and Statistics}}
+
+**DESCRIPTION (5-7+ sentences):**    
+{{Plot of Incident cases Vs. Date (automatically updated with data source)
+Plot of R number Vs. Date 
+Plot in Tableau (optional)
+Forecast of number of cases in the future (optional)}}
+
+### :pushpin: Step 2
+:family: **TARGET AUDIENCE (3-5+ sentences):**    
+{{colledge students who are interested in epidemics or data visualization}}
+
+### :pushpin: Step 3
+> Outline your learning/teaching structure:
+
+**Beginning (2-3+ sentences):**    
+{{import and prepare data}}
+
+**Middle (2-3+ sentences):**    
+{{plot data}}
+
+**End (2-3+ sentences):**    
+{{regression}}
```

Comment:
Same, with the Beginning, Middle, and End sections. We need descriptive, full sentences. This is to ensure you've atleast thought through a little bit about the concepts that will go into your blog and how to structure them. Vague subheadings don't really do much.